scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -21,8 +22,8 @@ enum location_types {
 }
 global.trait_list = {
 	"champion":{
-		weapon_skill : 20,
-		ballistic_skill:20,
+		weapon_skill : [10,5,"max"],
```

Comment:
what does this notation mean, [10, 5, "max"]?

Reply:
this notation can found in this https://docs.google.com/spreadsheets/d/1QD4yjUzIG1iMikTsrGGmBTHxUFLm7IpN-VaotICypVo/edit#gid=0 but i'll port into the main code basically i intend all traits to be imported in through csv. but this allows the traits to be randomised via a gauss method